### DISCUSSION TITLE
Temporarily Disable flux2

### DISCUSSION THREAD

**Discussion by SimplySeth** [fluxcd/flux2#1661] [Q&A]
In flux2 we no longer have disable via annotation. 

How do we disable flux2 from updating?

do I just add `--registry-disable-scanning ` to the deployment?

**Comment by jwerre** [comment]
You can use 
```
flux suspend image update flux-system
```
and to start it again:
```
flux resume image update flux-system
```

**Comment by yebyen** [comment]
The equivalent command is `flux suspend` which takes a parameter for the resource kind, and has an `--all` param too.

You can run

`flux suspend --all` to stop everything

https://fluxcd.io/docs/cmd/flux_suspend/

Or `flux suspend image update --all`

https://fluxcd.io/docs/cmd/flux_suspend_image/

This page covers this topic, under the subject of Incident Management:

https://fluxcd.io/docs/guides/image-update/#incident-management
